src/test/java/com/epam/tc/hw2/ex1/Exercise1.java

```diff
@@ -0,0 +1,156 @@
+        webElement.click();
+        webElement = webDriver.findElement(By.id("name"));
+        webElement.click();
+        webElement.sendKeys("Roman");
+        webElement = webDriver.findElement(By.id("password"));
+        webElement.click();
+        webElement.sendKeys("Jdi1234");
+        webElement = webDriver.findElement(By.id("login-button"));
+        webElement.click();
+
+        //        4. Assert Username is logged in
+
+        webElement = webDriver.findElement(By.xpath("//*[text()='Roman Iovlev']"));
+        String logNameResult = webElement.getText().toUpperCase();
+
+        assertThat(logNameResult)
+            .as("Username")
+            .isEqualTo("ROMAN IOVLEV");
+
+        //        5. Assert that there are 4 items on the header section are displayed
+        //        and they have proper texts
+
+        String[][] headerSectionArray = {
+            {"//*[text()='Home']", "HOME"},
+            {"//*[text()='Contact form']", "CONTACT FORM"},
+            {"//*[text()='Service']", "SERVICE"},
+            {"//*[text()='Metals & Colors']", "METALS & COLORS"}
+        };
+
+        for (String[] strings : headerSectionArray) {
+            webElement = webDriver.findElement(By.xpath(strings[0]));
+            String headerSectionText = webElement.getText().toUpperCase();
+            assertThat(headerSectionText)
+                .as("Text in the Header Section")
+                .isEqualTo(strings[1]);
+        }
+
+        //        6. Assert that there are 4 images on the Index Page and they are displayed
+
+        List<String> indexPageImages = List.of("icons-benefit", "icon-custom", "icon-multi", "icon-base");
+
+        for (String indexPageClassName : indexPageImages) {
+            webElement = webDriver.findElement(By.className(indexPageClassName));
+            boolean indexPagePicture = webElement.isDisplayed();
+            assertThat(indexPagePicture)
+                .as("Index page some picture")
+                .isTrue();
+        }
+
+        //        7. Assert that there are 4 texts on the Index Page under icons and they have proper text
+
+        List<WebElement> indexPageFourTexts = webDriver.findElements(By.className("benefit-txt"));
+        assertThat(indexPageFourTexts.get(0).getText()).isEqualTo("To include good practices\n"
+            + "and ideas from successful\n"
+            + "EPAM project");
+        assertThat(indexPageFourTexts.get(1).getText()).isEqualTo("To be flexible and\n"
+            + "customizable");
+        assertThat(indexPageFourTexts.get(2).getText()).isEqualTo("To be multiplatform");
+        assertThat(indexPageFourTexts.get(3).getText()).isEqualTo("Already have good base\n"
+            + "(about 20 internal and\n"
+            + "some external projects),\n"
+            + "wish to get more…");
+
+        //        8. Assert that there is the iframe with “Frame Button” exist
+
+        webElement = webDriver.findElement(By.id("frame"));
```

Comment:
If frame does not exist, some exception will be thrown
Try use wrong id to see

Reply:
Can you clarify what I have to do with it?